packages/common/src/ui/layout/tables/components/DataRow/DataRow.tsx

```diff
@@ -104,7 +104,9 @@ export const DataRow = <T extends DomainObject>({
             }}
             in={isExpanded}
           >
-            {ExpandContent ? <ExpandContent rowData={rowData} /> : null}
+            {ExpandContent && isExpanded ? (
```

Comment:
this doesn't work for me 🤔 The row doesn't expand in the outbound or inbound detail views

Reply:
oops 🙄 

Yup it didn't actually work 😁 

I tried a few things.. adding the `exitOnMount` and `enterOnMount` and `enter` props etc etc but couldn't get it to work. Then I realised that this isn't an issue for what this is fixing as this was essentially fixed by not having the whole table rerender..

So i've reverted the change which makes no difference to the modal performance. Though, it would be best to not render all the mini tables without being expanded